### DISCUSSION TITLE
ersem.com expired

### DISCUSSION THREAD

**Discussion by jornbr** [pmlmodelling/ersem#153] [General]
The domain registration for [ersem.com](https://ersem.com) seems to have expired. It now shows

![image](https://github.com/user-attachments/assets/71863bfb-e153-47ee-a411-b660957cb0b2)

Before it used to redirect to a PML page, I think https://www.pml.ac.uk/science/Projects/ERSEM-(European-Regional-Seas-Ecosystem-Model). Assuming this change is intentional, what is now the preferred way to link to ERSEM? We will then update links at https://fabm.net acccordingly.

**Comment by glessin** [comment]
The best way forward would be to point directly to the GitHub page https://github.com/pmlmodelling/ersem. Could you please update fabm.net links accordingly?
